docs/.vuepress/components/cdr-doc-tabs.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="cdr-doc-tabs">
+    <div class="cdr-doc-tabs__labels">
+      <ul class="cdr-doc-tabs-list">
+        <li class="cdr-doc-tabs__list-item" v-for="tab in tabLabelData" :key="tab.slug">
+          <a class="cdr-doc-tabs__link" :class="{ 'cdr-doc-tabs__link--active' : tab.active }" :href="tab.anchor" v-on:click="switchActiveTab(tab.label, $event)">
```

Comment:
This should also get an `aria-selected` boolean
and an ID

Reply:
It would be lovely if the focus could shift from the tab-list items to the first heading within the tab panel content on select

Reply:
@mhewson I've added the `aria` attributes. Thanks for suggesting those. I didn't try to manipulate the focus behavior, because if I want to keyboard navigate quickly through the tabs to skim the content, I'd want the focus to remain on the tab set, not jump down into the content. 

Is there an accessibility rationale I'm not seeing that would warrant that kind of automatic focus shift? I've seen it on modals, or in other cases of a context shift (off-canvas menu invocation) but not for tabs.